### DISCUSSION TITLE
I.D.E recommendation

### DISCUSSION THREAD

**Discussion by ghost** [community/community#145986] [New to GitHub] [answered]
What I.D.E is best for beginners?

**Comment by CyberSphinxxx** [comment, marked as answer]
[Visual Studio Code](https://code.visualstudio.com/)

**Code in any language**
VS Code supports almost every major programming language. Several ship in the box, like JavaScript, TypeScript, CSS, and HTML, but extensions for others can be found in the VS Code Marketplace.

**Comment by Sharath1036** [comment]
Most of the developers use VSCode
However, you can even use Cursor which is similar to VSCode and has a copilot like blackbox but with LLMs like Claude, GPT, etc in which you can chat with your code files.
